mm2src/mm2_main/src/database/my_swaps.rs

```diff
@@ -52,8 +52,12 @@ pub const TRADING_PROTO_UPGRADE_MIGRATION: &[&str] = &[
     "ALTER TABLE my_swaps ADD COLUMN taker_coin_nota BOOLEAN;",
 ];
 
+/// Adds Swap Protocol version column to `my_swaps` table
+pub const ADD_SWAP_VERSION_FIELD: &str = "ALTER TABLE my_swaps ADD COLUMN swap_version INTEGER;";
+/// Sets default value for `swap_version` to `1` for existing rows
+pub const SET_DEFAULT_SWAP_VERSION: &str = "UPDATE my_swaps SET swap_version = 1 WHERE swap_version IS NULL;";
```

Comment:
About naming vars:
I think SET_DEFAULT_SWAP_VERSION here actually means not 'default' but 'existing'.

Plus, there is also a comment in the code:
```
/// When a new [TakerOrderBuilder::new] is created, it sets [SWAP_VERSION] by default.
```
So I guess, SWAP_VERSION is in fact SWAP_VERSION_DEFAULT

Reply:
Thanks for the catch, fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2324/commits/731e605fc2b53b9f4afa9e13e3682c5be7ed9cf6